x-pack/test/functional_with_es_ssl/apps/cases/group1/view_case.ts

```diff
@@ -1013,11 +1013,22 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
     describe('Tabs', () => {
       createOneCaseBeforeDeleteAllAfter(getPageObject, getService);
 
+      it('renders tabs correctly', async () => {
+        await testSubjects.existOrFail('case-view-tab-title-activity');
+        await testSubjects.existOrFail('case-view-tab-title-files');
+        // there are no alerts in stack management yet
```

Comment:
When this PR https://github.com/elastic/kibana/pull/172217 is merged we can test the alerts table in the stack.